sdk/resourcemanager/azure-resourcemanager-cdn/pom.xml

```diff
@@ -124,6 +132,14 @@
           </excludes>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <failOnWarning>false</failOnWarning>
```

Comment:
No idea why it keeps failing even after `SuppressWarnings("unchecked")` is added. Added maven compiler suppresion here.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2663424&view=logs&jobId=33569338-7799-5183-558c-0111972ad234&j=33569338-7799-5183-558c-0111972ad234&t=24118881-f9d9-51a9-a03e-f3a022528ce7

Reply:
Which line is causing the problem?

I kind of want to avoid this in pom.

Reply:
maybe "unchecked conversion", not sure, IDE maybe able to help add the suppression?

Reply:
IDE suggested SuppressionWarnings("unchecked"). After the suppression, IDE no longer has warnings but maven still fails. 
I also tried "all" and no luck..

list of valid suppresions(eclipse's version):
https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.jdt.doc.user%2Ftasks%2Ftask-suppress_warnings.htm

Reply:
Maybe add this to class level.

Reply:
Local verify passed! Will push and let CI check. Strange, but thanks!